maintainer/check_codestyle.sh

```diff
@@ -0,0 +1,36 @@
+#!/bin/bash
+
+echo "check_codestyle.sh CODESTYLE=$1 TRAVIS_BRANCH=$2 TRAVIS_PULL_REQUEST=$3 TRAVIS_COMMIT=$4 TRAVIS_COMMIT_RANGE=$5"
+
+CODESTYLE=$1
+TRAVIS_BRANCH=$2
+TRAVIS_PULL_REQUEST=$3
+TRAVIS_COMMIT=$4
+TRAVIS_COMMIT_RANGE=$5
```

Comment:
I think the script inherit the environment, as `ARG` will export environment variables, so I guess we don't need the arguments!